styles/stylesheet1.css

```diff
@@ -0,0 +1,386 @@
+* {
+    box-sizing: border-box;
+  }
+
+h1, h2, h3, label, legend {
+  font-family: 'Cormorant Garamond', serif;
+}
+
+p, a {
+  font-family:  'Work Sans', serif;
+}
+
+header.head {
+    background-image: url(../images/header-image.jpg);
+    background-size: cover;
+    padding: 2em 4em 4em;
+    padding-bottom: 10em;
+    color: white;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.head-logo {
+  width: 33%;
+}
+
+header h1 {
+    color: #77ad3b;
+    font-style: italic;
+    font-weight: 500;
+    font-size: 175%;
+    margin-top: 0%;
+}
+
+.checkout-button, .checkout-button a {
+    float: right;
+    clear: both;
+    background-color: #77ad3b;
+    padding-top: .5em;
+    padding-bottom: .5em;
+    padding-left: 3em;
+    padding-right: 3em;
+    font-weight: 900;
+    color: white;
+    text-decoration: none;
+    font-size: 1em;
+    
+}
+
+nav.navigation {
+    background-color: lightgray;
+    padding-top: 1.5em;
+    padding-bottom: 1.5em;
+}
+
+nav a {
+    padding-right: .5em;
+    padding-left: .5em;
+    text-decoration: none;
+    font-weight: 900;
+    font-size: 1.2em;
+    color: #034177;
+}
+
+nav a:not(:last-child):after  {
+  content: '\2001' '/';
+}
+
+nav a#first-nav  {
+    padding-left: 9em;
+}
+
+.cart {
+  float: right;
+  font-weight: 900;
+  font-size: 1em;
+  line-height: 1.5;
+  padding-left: .5em;
+  text-align: right;
+}
+
+.cart img {
+  width: 10%;
+}
+
+.main-content {
+    width: 80%;
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 4em;
+}
+
+.about-adopt {
+    display: flex;
+}
+
+.about-adopt img {
+    padding: 1.25em; 
+    width: 75%;
+    height: 75%;
+    margin: auto;
+}
+
+h2.blue-title {
+    color: #034177;
+    font-size: 2.5em;
+    padding-left: .5em;
+}
+
+h3.blue-title {
+  color: #034177;
+  font-size: 1.75em;
+  font-weight: 900;
+  margin-top: 0em;
+  margin-bottom: .75em;
+}
+
+h3.blue-title-checkout {
+  color: #034177;
+  font-size: 1.75em;
+  font-weight: 900;
+  margin-top: 1.5em;
+}
+
+.about-adopt-text p {
+    font-size: 1.4em;
+    font-weight: 100;
+    line-height: 1.4;
+    padding-left: 1em;
+}
+
+div.gallery-item {
```

Comment:
I'd love for you to find another method of laying these cards out. Instead of float use css grid or flexbox